hooks/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Add a comment at the top of this file, or a README in this directory, to explain what this is doing here.  Is the name of this file hard-coded in some docker thing or can it be moved somewhere else?

Reply:
I added a README with a link to the supporting documentation since you can add other custom build phase hook files for Dockerhub.

Reply:
I would rather not have something like this in the root directory if we don't need it. How are you passing this variable from dockerhub and is there another way you could do it?